pywbem/_listener.py

```diff
@@ -1146,10 +1164,54 @@ def stop(self):
             self._https_thread = None
             self.logger.info("Stopped threaded HTTPS server")
 
-    def deliver_indication(self, indication, host):
+        self.logger.info("Stopping queued delivery")
+        self.delivery_stop = True
+        self.logger.info("Stopped threaded Deliver queue")
```

Comment:
Where is the stop of the queue thread?

Reply:
The stop is used to stopthe loop in deliver_indications_from_queue(...).

The better question is where is the thread kill.  Answer, same place as the kill for http and https threads, i.e. when the the whole python program stops.  This should be fixed together with the other threads issue since the change will be common to all of them.